### ISSUE TITLE
2.1.0 broken?

### ISSUE THREAD

**Issue by cduff** [JohannesKlauss/react-hotkeys-hook#252]
I upgraded from 1.6.1 to 2.1.0 and the hook has stopped working. Doesn't seem to get triggered by the keys any more.

I'm able to reproduce in a vanilla CRA TypeScript project.

_Note: Reproduced using Node 12.16.1 (LTS) on Windows 10._

```
npx create-react-app test --template typescript
cd test
npm i react-hotkeys-hook
```

Modify App.tsx to documented 'usage' example:

```
import React, { useState } from "react";
import { useHotkeys } from "react-hotkeys-hook";
function App() {
  const [count, setCount] = useState(0);
  useHotkeys("ctrl+m", () => setCount((prevCount) => prevCount + 1));
  return <div>Pressed {count} times.</div>;
}
export default App;
```

Run it and "ctrl+m" doesn't increment the count as expected.

If you do:
```
npm i react-hotkeys-hook@1.6.1
```
then it works as expected.

**Comment by cduff** [comment]
FYI I tested 2.0.1 and it works.

**Comment by JohannesKlauss** [comment]
@cduff Thx for your report, you are absolutely right. Must have slipped my mind. I fixed it in 2.1.1.
I guess since more and more people are using this package I should start to add tests to have a more concise and stable release process.
